Server/src/bgp/linkstate/MPLinkStateAttr.cpp

```diff
@@ -505,13 +592,49 @@ namespace bgp_msg {
                 break;
             }
             case ATTR_PREFIX_OSPF_FWD_ADDR:
-                //SELF_DEBUG("%s: bgp-ls: parsing prefix OSPF forwarding address attribute", peer_addr.c_str());
+                // SELF_DEBUG("%s: bgp-ls: parsing prefix OSPF forwarding address attribute", peer_addr.c_str());
                 LOG_INFO("%s: bgp-ls: prefix OSPF forwarding address attribute, not yet implemented", peer_addr.c_str());
                 break;
 
             case ATTR_PREFIX_OPAQUE_PREFIX:
                 LOG_INFO("%s: bgp-ls: opaque prefix attribute (len=%d), not yet implemented", peer_addr.c_str(), len);
                 break;
+                
+            case ATTR_PREFIX_SID_TLV: {
+                val_ss.str(std::string());
+                value_32bit = 0;
+                
+                // Package structure:
+                // https://tools.ietf.org/html/draft-ietf-ospf-segment-routing-extensions-09#section-5
+                
+                val_ss << this->parse_flags_to_string(*data, std::array<std::string, 8>{"", "NP", "M", "E", "V", "L"});
```

Comment:
Flags to header file. This will help us track and keep them consistent with documentation.